pkg/internal/feed/exporter_sql.go

```diff
@@ -201,6 +201,21 @@ func (e *SQLExporter) WriteMedia(auditID, mediaID, contentType string, body []by
 
 // NewSQLExporter creates a new instance of the SQLExporter
 func NewSQLExporter(dialect, connectionString string, autoMigrate bool, exportMediaPath string) (*SQLExporter, error) {
+	l, db, err := GetDatabase(dialect, connectionString)
+	if err != nil {
+		return nil, errors.Wrap(err, "connect to DB")
+	}
+
+	return &SQLExporter{
+		DB:              db,
+		Logger:          l,
+		AutoMigrate:     autoMigrate,
+		ExportMediaPath: exportMediaPath,
+		duration:        0,
+	}, nil
+}
+
+func GetDatabase(dialect string, connectionString string) (*zap.SugaredLogger, *gorm.DB, error) {
```

Comment:
this function should not return a logger

Reply:
yeah stupid me. goland refactoring isn't that smart after all. :P

Reply:
done.